decidim-proposals/config/locales/en.yml

```diff
@@ -527,7 +527,7 @@ en:
             create: Create
             title: Create proposal
           publish_answers:
-            number_of_proposals: Answers for %{number} proposals will be published.
+            number_of_proposals: Answers for %{number} proposal's will be published?
```

Comment:
Are you sure about this change?

Reply:
Have you attempted to see the change difference in the browser? For me the question mark at the end of the sentence seems approprate because the published answers give the user a choice (publish or cancel).

Just a minor improvement to the UX